internal/validations/clusterlogforwarder/validate_clusterlogforwarder_service_account.go

```diff
@@ -112,16 +115,17 @@ func gatherPipelineInputs(clf loggingv1.ClusterLogForwarder) sets.String {
 		}
 	}
 
+	noOfReceivers := 0
 	for _, input := range clf.Spec.Inputs {
 		if inputRefs.Has(input.Name) && input.Application != nil {
 			inputTypes.Insert(loggingv1.InputNameApplication)
 		}
 		if input.Receiver != nil && input.Receiver.HTTP != nil {
-			inputTypes.Insert(loggingv1.InputNameAudit)
+			noOfReceivers += 1
```

Comment:
Are we counting all receivers or just the HTTP ones?

Reply:
at this point we only have on type of receiver and we want to ensure http is configured.  Maybe it is good enough to simply check if receiver is non nil

Reply:
I'm wondering because of the syslog receiver that's coming up, how it should be treated.

Reply:
I figured that was the source of your inquiry.  There should be validation that ensures one or the other type is populated.  This means, in theory, checking that receiver is non-nil should be sufficient